app/assets/stylesheets/_design-system-waiting-room.scss

```diff
@@ -0,0 +1,15 @@
+// To be removed once design system incorporates styles included below.
```

Comment:
This file isn't currently being imported anywhere.

Reply:
Ah, it's created in #5944 - I'll be sure to wait for that to merge then resolve conflicts before merging mine.